NewForce-Cohort-6/tabloidfullstack-mashed-potatoes
# 24 user details

**Trello Ticket #24**

https://trello.com/c/2LemuKKQ/25-24-view-user-profile-details

Added functionality to view details of users, each with its display name, image (if available), full name, email and user type.

ApplicationViews.js - users Route added
UserDetails.js added

**Testing Instructions**

1.git add and git commit whatever you're working on on your branch
2. git checkout main
3 git fetch -a
4 git checkout 24UserDetails
5 If you don't see the most recent version of my files, git pull origin 24UserDetails
6 If you haven't created the Tabloid database, please do so.
Use this data to fill out tables and columns: [SQL Tables](https://github.com/NewForce-Cohort-6/tabloidfullstack-mashed-potatoes/blob/main/SQL/01_Tabloid_Create_DB.sql)
Use this to seed the database : [SQL SEED DATA](https://github.com/NewForce-Cohort-6/tabloidfullstack-mashed-potatoes/blob/main/SQL/02_Tabloid_Seed_Data.sql)
7 Click the execute with debugger solid green button to launch the app
8 Log into the website using “aotton2@ow.lyx” if greeted with a login page
9 From the home page, click the “User Profile” option in the navbar
10 The url should change to /users and a list of users should appear, with display name, full name and user type.
11 Display Names will be blue and clickable.
12. Once clicked on chosen user, you will be directed to User Details page with name, email, image, usertype.  
13. A 'Go Back' button will take you back to User List page.
